src/Symfony/Component/Routing/Tests/Matcher/Dumper/CompiledUrlMatcherDumperTest.php

```diff
@@ -442,21 +442,27 @@ public function getRouteCollections()
         $hostCollection->add('r1', (new Route('abc{foo}'))->setHost('{foo}.exampple.com'));
         $hostCollection->add('r2', (new Route('abc{foo}'))->setHost('{foo}.exampple.com'));
 
+        /* test case 14 */
+        $fixedLocaleCollection = new RouteCollection();
+        $fixedLocaleCollection->add('home.fr', new Route('/{_locale}/accueil', ['_locale' => 'fr', '_canonical_route' => 'home'], ['_locale' => 'fr']));
```

Comment:
Can we use the localized feature instead of writing routes by hand? This will ensure it integrates effectively with the logic.

Reply:
Unfortunately, that feature comes with de74794acfceb1d793d0a7686caaed6d3f5ae9a5 which isn't in 4.4

Reply:
Localized routes exist since 4.1, there must be a way :)

Reply:
Agreed, I'll search :)